server/routes/api/choropleth.py

```diff
@@ -35,52 +35,70 @@
   url_prefix='/tools/choropleth'
 )
 
-@bp.route('/api')
-def choropleth_api():
+@bp.route('/values')
+def choropleth_values():
     """Returns data for geographic subregions for a certain statistical 
             variable.
 
-    API Params: #TODO(iancostello): Split API calls up or move to client.
-        statVar -> The statistical variable to download, as a string.
-        perCapita -> Whether to return the per-capita value, as a string of a 
-                boolean.
-        geoId -> The currently viewed geography to render, as a string.
-        level -> The subgeographic level to pull and display information for,
+    API Params:
+        geoId: The currently viewed geography to render, as a string.
```

Comment:
nit: please use dcid to standardize with our other API's (also would apply better to non-US places)

Reply:
changed